MoreInfo.jsx

```diff
@@ -0,0 +1,108 @@
+import React, { useState, useEffect, useContext } from "react";
+import { useLocation } from "react-router";
+import AuthContext from "../../context/AuthProvider";
+import { getAllBookings } from "../../../Utils/DataManagement";
+import useLocalStorage from "../../../Hooks/useLocalStorage";
+import NavBar from "../../NavBar/NavBar";
+import { parseISO, isSameDay, format } from "date-fns";
+import { faPen, faTrashCan } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import UpdateTrainingForm from "./UpdateTrainingForm/UpdateTrainingForm";
+import DeleteTraining from "./DeleteTraining/DeleteTraining";
+import { NO_BOOKINGS, PEOPLE_WHO_BOOKED } from "../../../Utils/StringLiterals";
+import "./MoreInfo.css";
+
+export default function MoreInfo({ booking }) {
+  const location = useLocation();
+  const [selectedDay, setSelectedDay] = useState(location?.state?.selectedDay);
```

Comment:
I'm wondering what location?.state? is for?  why selectedDay and training is under location?

Reply:
I passed selectedDay and training using useNavigate from another component, that's why I have that location?.state?